### PR TITLE
feat(tag)

### PR DESCRIPTION
**Pull request by ccedrone**

### CODE REVIEW

**Review by orion-cengage — CHANGES_REQUESTED** [review]
Please make the following changes:

1. I'm not seeing a section on the API docs page for the small sizes

2. Change the labels on the examples to just: Default, Primary, Low Contrast, High Contrast
image

3. Change labels on examples to just: Disabled, Disabled Low Contrast
image

4. Change labels on examples to just: Inverse Default, Inverse Primary, Inverse Low Contrast, Inverse High Contrast
image

5. Change labels on examples to just: Inverse Disabled, Inverse Disabled Low Contrast
image

6. I think the titles of these sections should be "Tag With Icon" and "Deletable Tag With Icon"
image
image

7. Small tag with delete button: The right margin on the close button icon should be 4px, not 2px
image

**Comment by orion-cengage** [comment]
@ccedrone I forgot one update to make.  The API Docs page needs the button in the upper-right that goes to the Design Guidelines page.

File: packages/react-magma-dom/src/i18n/default.ts [hunk 1]
```diff
@@ -225,6 +225,9 @@ export const defaultI18n: I18nInterface = {
     nextButtonLabel: 'Scroll tabs forward',
     previousButtonLabel: 'Scroll tabs back',
   },
+  tag: {
+    deleteAriaLabel: 'Delete tag',
```
Comment (rcgillet) [inline packages/react-magma-dom/src/i18n/default.ts:229]: This should be more descriptive. We do something in `Combobox` to use the `labelText`.

`'reset selection for {labelText}. {selectedItem} is selected',`. So here we could do `delete {labelText} tag`. Then in the `Tag`  component file you can do something like this ```const deleteAriaLabel = i18n.tag.deleteAriaLabel.replace(/\{labelText\}/g, labelText)```

File: website/react-magma-docs/src/pages/api/tag.mdx [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    </I18nContext.Provider>
+  );
+}
+```
+
+## Tag Props
+
+**Any other props supplied will be provided to the wrapping `div` element.**
+
+<DeleteableTagProps />
```
Comment (rcgillet) [inline website/react-magma-docs/src/pages/api/tag.mdx:306]: Above each of the `props` tables put `### Deletable Tag Props` or `### Clickable Tag Props` so they know what they are looking at.

**Review by rcgillet — CHANGES_REQUESTED** [review]

**Review by orion-cengage — APPROVED** [review]
Styling is looking good.

**Review by rcgillet — APPROVED** [review]

**Comment by ccedrone** [comment]
In the development of this component, pimaryInverse was needed. This ticket was resolved as part of the work: https://github.com/cengage/react-magma/issues/260

**Pull request merged**